src/app/app-routing.module.ts

```diff
@@ -1,104 +1,123 @@
-import { RoutingPublicAuthService } from './api/services/routing-public-auth.service';
-import { RoutingPrivateAuthService } from './api/services/routing-private-auth.service';
 import { Routes, RouterModule } from '@angular/router';
 import { NgModule } from '@angular/core';
 import { MainCoreLegacyComponent } from './view/components/main-core-legacy/main-core-legacy-component';
 import { MainComponentLegacy } from './view/components/main-legacy/main-legacy.component';
 import { LoginPageComponent } from './view/components/login/login-page-component';
 import { LogOutContainer } from './view/components/login/login-component/log-out-container';
 import { IframePortletLegacyComponent } from './view/components/_common/iframe/iframe-porlet-legacy/index';
-import { RoutingContentletAuthService } from './api/services/routing-contentlet-auth.service';
+import { AuthGuardService } from './api/services/guards/auth-guard.service';
+import { ContentletGuardService} from './api/services/guards/contentlet-guard.service';
+import { MenuGuardService } from './api/services/guards/menu-guard.service';
+import { PublicAuthGuardService } from './api/services/guards/public-auth-guard.service';
 
-const PORTLETS_ANGULAR: Routes = [
+const AUTH_MODULES: Routes = [
+    {
+        path: 'forgotPassword',
+        loadChildren:
+            'app/view/components/login/forgot-password-component/forgot-password.module#ForgotPasswordModule'
+    },
+    {
+        path: 'login',
+        loadChildren: 'app/view/components/login/login-component/login.module#LoginModule'
+    },
+    {
+        path: 'resetPassword/:token',
+        loadChildren:
+            'app/view/components/login/reset-password-component/reset-password.module#ResetPasswordModule'
+    },
+    {
+        path: '',
+        children: []
```

Comment:
We talked about a ```redirecTo``` instead of the children empty... didn't work?

Reply:
redirecTo make a new route change, so it goes over routing just to determinate that the route is invalid, with this solution the guard capture that is an invalid route and do the  desired behavior.